luigi/static/visualiser/index.html

```diff
@@ -21,11 +21,10 @@
         <script src="lib/jquery.slimscroll.min.js"></script>
         <script src="lib/AdminLTE/js/app.min.js"></script>
         <script src="lib/datatables/js/jquery.dataTables.min.js"></script>
-        <script src="lib/URI.js/1.18.2/URI.js"></script>
+        <script src="lib/URI.js"></script>
```

Comment:
In setup.py it copies the static files with this function:

def get_static_files(path):
    return [os.path.join(dirpath.replace("luigi/", ""), ext)
            for (dirpath, dirnames, filenames) in os.walk(path)
            for ext in ["*.html", "*.js", "*.css", "*.png",
                        "*.eot", "*.svg", "*.ttf", "*.woff", "*.woff2"]]


The URI folder was called URI.js, and the above function fails when it gets this folder extension